backend/app/models/inquiry.py

```diff
@@ -17,29 +20,44 @@
 # Shared properties
 class InquiryBase(SQLModel):
     text: str = Field(min_length=MIN_LENGTH, max_length=MAX_LENGTH)
+    theme_id: int | None = Field(
+        foreign_key="theme.id", ondelete="CASCADE", nullable=True
+    )
 
 
 # Properties to receive on inquiry creation
 class InquiryCreate(InquiryBase):
     text: str = Field(min_length=MIN_LENGTH, max_length=MAX_LENGTH)
 
 
+# Properties to receive on inquiry update
+class InquiryUpdate(InquiryBase):
+    id: int
+
+
+# Properties to receive on inquiry delete
+class InquiryDelete(InquiryBase):
+    id: int
+
+
 # Database model, database table inferred from class name
 class Inquiry(InquiryBase, IdMixin, table=True):
     text: str = Field(min_length=MIN_LENGTH, max_length=MAX_LENGTH, unique=True)
     created_at: datetime = Field(default_factory=lambda: datetime.now(timezone.utc))
-
-    scheduled_inquiries: list["ScheduledInquiry"] = Relationship(
+    scheduled_inquiry: "ScheduledInquiry" = Relationship(
```

Comment:
I changed ScheduledInquiry to be a one-to-one relationship with Inquiry because that seems to be how it will work. Please correct me if I am wrong. Other than adding a unique constraint on scheduled_inquiry.inquiry_id the DB structure remains the same as before.